### ISSUE TITLE
Mimosa26 automatic threshold tuning and column masking

### ISSUE THREAD

**Issue by laborleben** [SiLab-Bonn/pymosa#10]
- [ ] global threshold tuning (register IVDREF2)
- [ ] quarter threshold tuning (registers IVDREF1A/B/C/D)
- [ ] noisy column masking (register DisableLatch)

**Comment by laborleben** [comment]
Maybe we add a standard config file where we set the threshold to 255 (or whatever is the maximum) and which we load from beginning and afterwards just write the threshold values and other important registers using the ```set_configuration()``` method provided by basil.

**Comment by YannickDieter** [comment]
Yes, 255 is max. Good idea, then we can use this as a starting point for the tuning, a modify this, load it and then scan,.. and so on.

**Comment by YannickDieter** [comment]
Upto now only the local registers IVDREF1A/B/C/D are tuned. Global threshold is not touched and no noisy pixel masking is implemented. See PR #19.
